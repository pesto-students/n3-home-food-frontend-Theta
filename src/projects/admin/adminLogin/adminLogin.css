.left-container{
    height: 100vh;
    background:#000000d9;
    display: flex;
    align-items: center;
   }
   .left-container-content{
       padding: 40px;
       width: 80%;
       height: auto;
       color: #fff;
   }
   .left-container-content h1{
       border-bottom: 2px solid;
       padding-bottom: 26px;
       color: #fff;
       font-family: system-ui;
       width: 70%;
   }
   .right-container{
       padding: 20px 50px;

    height: 100vh;
    display: flex;
    flex-direction: column;
    justify-content: center;


   }
   
.right-container p{
text-align: left;
font-weight: 700;

}
