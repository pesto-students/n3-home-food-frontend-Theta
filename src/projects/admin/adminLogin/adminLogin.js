import { Col, Row } from "antd";

import { React } from "react";
import { connect } from "react-redux";
import Login from "components/login/login";
import "./adminLogin.css";
import { useTranslation } from "react-i18next";

function AdminLogin() {
  const { t } = useTranslation();
  return (
    <>
      <Row style={{ height: "100vh" }}>
        <Col md={16} className="left-container">
          <div className="left-container-content">
            <h1>{t("Admin.headLine")}</h1>
            <p>{t("Admin.headDescription")}</p>
          </div>
        </Col>
        <Col md={8}>
          <div className="right-container">
            <p>{t("Landing.Login")}</p>
            <Login userType="Admin" />
          </div>
        </Col>
      </Row>
    </>
  );
}

const mapStateToProps = (state) => {
  return {
    title: state,
  };
};
export default connect(mapStateToProps)(AdminLogin);
