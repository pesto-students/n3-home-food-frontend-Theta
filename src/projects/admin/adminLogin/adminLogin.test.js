import * as React from "react";
import Enzyme, { shallow } from "enzyme";
import EnzymeAdapter from "enzyme-adapter-react-16";
import adminLogin from "./adminLogin";

Enzyme.configure({ adapter: new EnzymeAdapter() });

describe("Admin Login component", () => {
  let wrapper;
  test("render without error", () => {
    wrapper = shallow(<adminLogin />);
    expect(wrapper.length).toBe(1);
  });

  test('should render correctly in "debug" mode', () => {
    const component = shallow(<adminLogin />);

    expect(component).toMatchSnapshot();
  });
});