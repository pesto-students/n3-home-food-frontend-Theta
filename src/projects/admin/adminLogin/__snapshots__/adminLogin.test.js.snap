// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Login component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
